packages/x-data-grid-premium/src/models/dataGridPremiumProps.ts

```diff
@@ -124,6 +129,11 @@ export interface DataGridPremiumPropsWithDefaultValue<R extends GridValidRowMode
    * @default (pastedText) => { const text = pastedText.replace(/\r?\n$/, ''); return text.split(/\r\n|\n|\r/).map((row) => row.split('\t')); }
    */
   splitClipboardPastedText: (text: string) => string[][] | null;
+  /**
+   * If `true`, the AI Assistant is enabled.
+   * @default false
+   */
+  enableAiAssistant: boolean;
```

Comment:
Maybe follow the same pattern as other feature flags (`${featureName}` or `disable${featureName}`)?
```suggestion
  aiAssistant: boolean;
```